includes/alerts.inc.php

```diff
@@ -160,13 +160,14 @@ function RunRules($device)
         return false;
     }
     foreach (GetRules($device) as $rule) {
-        echo " #".$rule['id'].":";
+        c_echo('Rule %p#'.$rule['id'].' (' . $rule['name'] . '):%n ');
         $inv = json_decode($rule['extra'], true);
         if (isset($inv['invert'])) {
             $inv = (bool) $inv['invert'];
         } else {
             $inv = false;
         }
+        d_echo(PHP_EOL);
```

Comment:
It's done to not screw the formatting up. You'll see without debug the output is all on one line, with debug it's broken down so I only wanted a newline for debug.